### PR TITLE
systemd-tmpfiles: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-tmp-files.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# systemd-mount
```
Comment (pixelcmtd) [inline pages/linux/systemd-tmp-files.md:1]: The file name `systemd-tmp-files.md` is also incorrect. Can you rename it to `systemd-tmpfiles.md`?

```suggestion
# systemd-tmpfiles
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/systemd-tmpfiles.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Clean up files and directories with age parameters configured:
+
+`systemd-tmpfiles --clean`
+
+- Remove files and directories as specified in the configuration:
+
+`systemd-tmpfiles --remove`
+
+- Apply operations for user-specific configurations::
```
Comment (acuteenvy) [inline pages/linux/systemd-tmpfiles.md:18]:
```suggestion
- Apply operations for user-specific configurations:
```

File: pages/linux/systemd-tmpfiles.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# systemd-tmpfiles
+
+> Create, delete and clean up volatile and temporary files and directories.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-tmpfiles.html>.
```
Comment (acuteenvy) [inline pages/linux/systemd-tmpfiles.md:4]:
```suggestion
> Create, delete and clean up volatile and temporary files and directories.
> This command is automatically invoked on boot by systemd services, and running it manually is usually not needed.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-tmpfiles.html>.
```

File: pages/linux/systemd-tmpfiles.md (hunk 2 above)
Comment (kbdharun) [reply pages/linux/systemd-tmpfiles.md:18]: I think we should add `::` detection to tldr-lint, this issue seems to be occurring pretty often and sometimes goes through our oversight too.

Comment (quantumflo) [reply pages/linux/systemd-tmpfiles.md]: @kbdharun I can work on it if there is an issue created

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**